tests/src/test/resources/kg/context/bbp-neuroshapes.json

```diff
@@ -2,7 +2,6 @@
   "@context": [
     "https://neuroshapes.org",
     {
-      "@base": "https://bbp.epfl.ch/neurosciencegraph/data/",
```

Comment:
When setting a base, it attempts to compact the ids relatively to this value by default.
It was enabled in the previous code but did not work for some reason.